pkg/yurttunnel/iptables/iptables.go

```diff
@@ -17,14 +17,62 @@ limitations under the License.
 package iptables
 
 import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/informers"
+	coreinformer "k8s.io/client-go/informers/core/v1"
 	clientset "k8s.io/client-go/kubernetes"
-	corelisters "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog"
+	utildbus "k8s.io/kubernetes/pkg/util/dbus"
 	"k8s.io/kubernetes/pkg/util/iptables"
 	"k8s.io/utils/exec"
+	utilnet "k8s.io/utils/net"
+
+	"github.com/alibaba/openyurt/pkg/yurttunnel/constants"
+)
+
+const (
+	kubeletSecurePort         = "10255"
+	kubeletInsecurePort       = "10250"
+	loopbackAddr              = "127.0.0.1"
+	reqReturnComment          = "return request to access node directly"
+	dnatToTunnelComment       = "dnat to tunnel for access node"
+	yurttunnelServerPortChain = "TUNNEL-PORT"
+	yurttunnelPortChainPrefix = "TUNNEL-PORT-"
+	defaultSyncPeriod         = 15
+
+	// constants related dnat rules configmap
+	yurttunnelServerDnatConfigMapName = "kube-system"
+	yurttunnelServerDnatConfigMapNs   = "yurttunnel-server-dnat-cfg"
```

Comment:
the name of configmap should keep consistency with old version

Reply:
initialize the variable based on the user input